redaxo/src/core/lib/error_sandbox.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * @package redaxo\core
+ * @internal
+ */
+class rex_error_sandbox
+{
+    private static ?string $error = null;
+
+    public static function run(callable $cb): mixed {
```

Comment:
Ich würd hier in der sandbox überall das static wegnehmen

Reply:
fixed in [`a50f3f2` (#5840)](https://github.com/redaxo/redaxo/pull/5840/commits/a50f3f25750cf8f211e98cf139c6908428bca88b)